server/metanome/config.yaml

```diff
@@ -0,0 +1,9 @@
+---
+files:
+  - input.csv
+config:
+  header: true
+  double_quote: true
+  quoting: true
+  delimiter: 44
+  escape:
```

Comment:
kann dass hier evtl. weggelassen werden, wenn da gar nichts drinne ist?

Reply:
die config-datei wird vom discovery-algorithmus benötigt, die muss man als Kommandozeilenargument übergeben. Passiert aber in der .JAR-Datei, daher sieht man das in diesem Repository nicht

Reply:
Ah ich meinte hier eigentlich nur das escape Argument? Ist ja aber auch nicht schlimm wenn es drinne ist.